docs/tutorials/openvino-mlflow.md

```diff
@@ -0,0 +1,611 @@
+|              |                       |         |      |                               |                                           |                            |
+|              |                       |         |      |                               |                                           |                            |
+| ovms         | openvino-model-server | 0.1.0   |      | [ API ]                       | [ API ]                                   |                            |
+|              |                       |         |      | resource: ovms-operator       | internal: http://kubernetes.default.svc   |                            |
+|              |                       |         |      | category: prediction-serving  |                                           |                            |
+|              |                       |         |      |                               |                                           |                            |
+|              |                       |         |      |                               |                                           |                            |
+| seldon-core  | seldon-core           | 1.11.0  |      | [ API ]                       | [ API ]                                   |                            |
+|              |                       |         |      | resource: seldon-core-api     | internal: http://kubernetes.default.svc   |                            |
+|              |                       |         |      | category: prediction-serving  |                                           |                            |
+|              |                       |         |      |                               |                                           |                            |
+|              |                       |         |      |                               |                                           |                            |
++--------------+-----------------------+---------+------+-------------------------------+-------------------------------------------+----------------------------+
+```
+
+## Training & Serving the Model
+
+We will be training a
+[Convolutional Neural Network (CNN)](https://developers.google.com/machine-learning/glossary/#convolutional_neural_network)
+to classify [CIFAR-10 images](https://www.cs.toronto.edu/~kriz/cifar.html) using the Keras Sequential API.
+The complete original code for the model training is available [here](https://github.com/fuseml/examples/tree/main/codesets/mlflow/keras). Our model architecture has been changed slightly, to make it converge faster and to yield better results.
+For an overview of the changes, please refer to [this Machine Learning Mastery article](https://machinelearningmastery.com/how-to-develop-a-cnn-from-scratch-for-cifar-10-photo-classification/).
```

Comment:
It's actually not clear from this paragraph what changes are you mentioning. _Our model architecture has been changed slightly_ : compared to what exactly?

Reply:
Ok, I rephrased it, hopefully it makes more sense now.